product_docs/docs/eprs/7/05_smr_operation/08_optimizing_performance/02_optimize_sync_replication/using_bacth_lob.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: "Using batch synchronization for LOB objects"
+---
+
+When synchronizing tables with LOB-type columns (with BLOB, CLOB, BYTEA etc. objects), the Replication Server copies the rows by grouping 5 rows in a batch by default. This is to avoid causing out-of-memory errors when an LOB column has a large amount of data. You can customize the batch size to increase or decrease the number of rows copied in a batch.
+
+To improve the performance of synchronization procedures, reduce network roundtrips and speed up data replication, EDB supports copying rows with LOBs in larger batches. This way, the Replication Server can copy several rows at the same time, making the synchronization of tables faster. EDB recommends configuring larger batches only if the instance hosting the replication server has enough memory.
```

Comment:
Since you already told us that you can customize the batch size to increase or decrease the number of rows copied in a batch, I think you just need to note the option here like "To improve the performance of synchronization procedures, reduce network roundtrips and speed up data replication, you can copy rows with LOBs in larger batches."